homeassistant/components/life360/services.yaml

```diff
@@ -0,0 +1,2 @@
+zones_from_places:
```

Comment:
Why is this a service?

Reply:
If a user doesn't update Places in Life360 very often, or maybe never at all, then HA zones can be created/updated just once at startup. However, if/when they add, remove or change Places in Life360, they need a way to update HA accordingly. I suppose that could be done by forcing the user to restart HA. Rather than require a restart I provided two optional, alternative ways: they can set up HA to periodically poll the Life360 Places and update HA accordingly. Since this is something that probably doesn't happen very often it's probably best to set a fairly large poll period. Of course, that means when Places _are_ changed it will take a while before HA updates. The other method is via a service. This way the user can update Life360 Places, then update HA immediately. Which way a user decides to go is based on their needs and inclinations.